docs/api-docs/getting-started/webhooks/about-webhooks.md

```diff
@@ -153,9 +153,9 @@ After the final retry attempt (cumulatively **48 hours** after the first deliver
 
 To ensure webhook callback requests are secure, BigCommerce takes the following precautions:
 
-
 * Webhook payloads contain minimal information about the store and event
 * Webhook payloads are sent over **TLS-encrypted** connection
+* Registered webhooks are automatically deactivated in our system on app uninstall

```

Comment:
I don't think this should go in the **Security** section, since its not specifically related to security.  I think this should go in a new section called **Deactivating Webhooks**. @amckemie -- what do you think?

Reply:
@aglensmith @traci707 I'm not sure it's worth creating a whole new section, since there is nothing the partner needs to do. We just need all app developers to know that when their app is uninstalled, BigCommerce will delete (I believe, but confirming with Andrii Lenets (who I can't tag here)). If we did other post-uninstall actions, I think having a section would make more sense. 

That said, maybe for navigation and ease of discoverability, having a section does make more sense? But rather than "Deactivating webhooks", something along the lines of "Post App Uninstall Actions" or "Handling Webhooks after App Uninstall".

Reply:
I made changes based on your feedback, take a look at the updated information below.